frontend/src/components/Table/MoneyCell.tsx

```diff
@@ -4,20 +4,38 @@ import { FastCurrencyInput } from 'components/common/form/FastCurrencyInput';
 import { IProperty } from 'actions/parcelsActions';
 import { CellProps } from 'react-table';
 import { formatMoney } from 'utils';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import TooltipWrapper from 'components/common/TooltipWrapper';
 
-export const EditableMoneyCell: React.FC<any> = ({
+interface IEditableCellProps {
+  namespace: string;
+  suppressValidation: boolean;
+  cell: CellProps<IProperty, number>;
+}
+
+export const EditableMoneyCell = ({
   namespace = 'properties',
   suppressValidation,
   cell,
-  ...props
-}: any) => {
+}: IEditableCellProps) => {
   const context = useFormikContext();
-  return (
+
+  const property = cell.row.original;
+  const canEdit = useKeycloakWrapper().canUserEditProperty(property);
+
+  return canEdit ? (
     <FastCurrencyInput
       formikProps={context}
       suppressValidation={suppressValidation}
       field={`${namespace}.${cell.row.id}.${cell.column.id}`}
     ></FastCurrencyInput>
+  ) : (
+    <TooltipWrapper
```

Comment:
I didn't really have design for this, but just displaying a read only row in the middle of a whole bunch of editable rows with no warning or anything seemed like bad UE, so hopefully this provides a minimal amount of feedback.